scripts/generate-renovate-changeset.ts

```diff
@@ -1,23 +0,0 @@
-// This script is designed so that Renovate can automatically add changesets to the PRs it
-// creates. It is designed to be run in the root of the repo.
```

Comment:
Why are we removing this?

Reply:
Unfortunately it doesn't work anymore due to internal security compliance. We'll need to come up with a different solution

Reply:
so not strictly related to this PR, but something to do anyway.